src/views/ForgotPassword.vue

```diff
@@ -47,13 +55,13 @@ export default {
     return {
       loading: false,
       model: {
-        email: ''
+        email: '',
       },
     };
   },
   methods: {
     async handleSubmit() {
-      const valid = await this.$refs.form.validate()
+      const valid = await this.$refs.form.validate();
 
       if (valid) {
         try {
```

Comment:
Try to avoid nested blocks.
In this case you can simply write:

`if (!valid) {
return
}

        try {
          this.loading = true;
          await authService.forgotPassword(this.model);
        } catch (error) {
          console.log(error);
        } finally {
          this.loading = false;
        }
      `